mountpoint-s3/src/main.rs

```diff
@@ -495,3 +507,9 @@ pub fn retrieve_instance_type() -> anyhow::Result<String> {
     let result = futures::executor::block_on(query).context("IMDS query failed")?;
     Ok(result)
 }
+
+pub fn get_maximum_network_throughtput(ec2_instance_type: &str) -> Option<f64> {
+    let file = include_str!("../scripts/network_performance.json");
+    let throughput: serde_json::Value = serde_json::from_str(file).expect("json file is malformed.");
+    throughput[ec2_instance_type].as_f64()
```

Comment:
Yeah, I guess it's OK if anyone building this project know that the `network_performance.json` must exist and valid. In that case, I'm happy with just adding some tests and updating error message to be more specific so that we know what to do when we get this error.